lms/djangoapps/discussion_api/tests/test_serializers.py

```diff
@@ -402,15 +407,20 @@ def test_children(self):
 
 
 @ddt.ddt
-class ThreadSerializerDeserializationTest(CommentsServiceMockMixin, UrlResetMixin, ModuleStoreTestCase):
+class ThreadSerializerDeserializationTest(CommentsServiceMockMixin, UrlResetMixin, SharedModuleStoreTestCase):
     """Tests for ThreadSerializer deserialization."""
+    @classmethod
+    @mock.patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
+    def setUpClass(cls):
+        super(ThreadSerializerDeserializationTest, cls).setUpClass()
+        cls.course = CourseFactory()
```

Comment:
Huh. I don't actually know, that wasn't intentional. I'm kind of surprised that didn't cause anything to barf. I'll change it back to `.create()`.